docs/Documentation/Development/Languages/Julia/julia_install.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Installing Julia
+postdate: November 7, 2022
+layout: default
+author: Jonathan Maack
+description: Guide to installing Julia on on NREL systems
+parent: Julia
+grand_parent: Languages
+---
+
+# Installation
+
+Julia modules exist on Eagle and provide an easy way to use it on the HPC system. Access simply with
```

Comment:
I think a version table should be added to show which modules are available on the other systems as well. The Applications template and the MPI PR have some examples of version tables.

The Running Julia section in Swift has some info that could be incorporated here.